examples/cancel.rs

```diff
@@ -9,7 +9,7 @@ use std::sync::{Arc, Mutex};
 use std::time::Duration;
 
 fn main() {
-    println!("Starting bars...");
+    eprintln!("Starting bars...");
```

Comment:
These can stay as `println!`, they're just demos after all.

Reply:
Is there any reason to keep them that way? `eprintln` seems more consistent to me, but I'll defer to your judgement.